tests/unit/htdocs/class/CriteriaTest.php

```
@@ -0,0 +1,1829 @@
+    public function testRenderFloatValueIsQuoted(): void
+    {
+        $c = new Criteria('price', 19.99);
+
+        $result = $c->render($this->db);
+        // Float values are NOT integers, so they get quoted as strings
+        $this->assertStringContainsString("'19.99'", $result);
+    }
+
+    public function testRenderLeadingZeroStringIsQuoted(): void
+    {
+        $c = new Criteria('zipcode', '01234');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)